lib/views/invoice_slack_view.ex

```diff
@@ -0,0 +1,29 @@
+defmodule Aprb.Views.InvoiceSlackView do
+  import Aprb.ViewHelper
+
+  def render(event) do
+    partner_data = fetch_partner_data(event["properties"]["partner_id"])
+    %{
+      text: ":money_with_wings: Invoice #{event["object"]["display"]} #{event["verb"]}",
+      attachments: "[{
```

Comment:
There's a lot of handwaving with my snippet. I think the second line would actually return something like `{ :ok, "[\"I'mmajson\"}]" }`. Happy to pair on it!